src/js/reducers/search.js

```diff
@@ -0,0 +1,23 @@
+import { SEARCHBY } from '../actions/actionTypes';
+import defaultState from '../default-state/';
+
+function search ( state = defaultState.search, action) {
+  switch (action.type) {
+    case 'SEARCHBY':
+      state = {
```

Comment:
For all those returns from reducers you can return just modified object withoud assigning
```
return {
        ...state,
        searchby: action.searchby
      }
```